scala/logback/README.md

```diff
@@ -0,0 +1,10 @@
+# Sentry Scala + Logback Example
+
+This project contains an integration example written in Scala and uses Sentry's Logback SDK.
+
+
+## Installation and Configuration
+1. Add the Sentry Logback package as dependency to the [build.sbt](https://github.com/getsentry/examples/blob/scala-logback/java/scala-logback/build.sbt#L8) file.
+2. Add the your DSN key in the [main function ](https://github.com/getsentry/examples/blob/scala-logback/java/scala-logback/src/main/scala/HelloWorld.scala#L5).
```

Comment:
One last thing, these URLs have to be fixed (they won't work until we merge though):

Prefix will be `https://github.com/getsentry/examples/blob/master/scala/logback/`...

Reply:
Fixed. Thanks a lot, Brett!